spec/calculator_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'calculator'
+
+describe Calculator do
+  subject(:calculator) { described_class.new }
+  let(:takeaway) { double :takeaway }
+  let(:menu) { {ribs: 3,
+        beef: 4,
+        rolls: 3,
+        chips: 2,
+        pies: 5}
+        }
```

Comment:
Lint/BlockAlignment: `}` at 11, 8 is not aligned with `let(:menu) { {ribs: 3,` at 6, 2.